packages/analytics/amplify_analytics_pinpoint_dart/tool/patch/user-models.patch

```diff
@@ -0,0 +1,787 @@
+From 173f432f1c7916d2a591394eda35f044a6bcad08 Mon Sep 17 00:00:00 2001
```

Comment:
What is the purpose of this?

Reply:
Not sure, looking at the full file it seems like it was related to a codegen update that Dillon had made: 

From: Dillon Nys <nydillon@amazon.com>
Date: Mon, 18 Jul 2022 13:13:49 -0700
Subject: [PATCH] Add user profile models

I think it was accidentally committed so I'm going to delete it.